R/solGS/combine_populations.r

```diff
@@ -4,13 +4,13 @@
 
 options(echo = FALSE)
 
-library(stats)
 library(stringr)
 library(randomForest)
-library(plyr)
 library(lme4)
 library(data.table)
 library(phenoAnalysis)
+library(dplyr)
+library(tibble)
```

Comment:
is tibble is the R deps?

Reply:
Yes, it is. But it is already installed in the systems; it is a dependency of another package that is listed in the R files.